File: Server/CaffStoreServer.WebApi/Models/Responses/CAFFResponse.cs
using System;

namespace CaffStoreServer.WebApi.Models.Responses
{
    public class CAFFResponse
    {
        public long Id { get; set; }

        public string Name { get; set; }

        public string CreationDate { get; set; }

        public string Creator { get; set; }

        public int Duration { get; set; }

        public string ThumbnailUrl { get; set; }

        public double Cost { get; set; }

        public bool Bought { get; set; }

        public string ImageUrl { get; set; }
    }
}

File: Android/app/src/main/java/com/itsecurityteam/caffstore/model/responses/CaffResponse.kt
package com.itsecurityteam.caffstore.model.responses

import java.time.LocalDateTime

data class CaffResponse(
    val id: Long,
    val name: String,
    val creationDate: LocalDateTime,
    val creator: String,
    val duration: Int,
    val thumbnailUrl: String,
    val cost: Double,
    var bought: Boolean,
    var imageUrl: String
)